src/main/webapp/js/instructorFeedbackResultsAjaxByQuestion.js

```diff
@@ -50,11 +51,23 @@ $(document).ready(function() {
                 $(panelHeading).click(toggleSingleCollapse);
                 $(panelHeading).trigger('click');
 
+                if (isPanelSetAsEmptyByBackend($panelBody)) {
+                    displayAsEmptyPanel($panelBody);
+                }
+
                 showHideStats();
             }
         });
     };
 
+    var isPanelSetAsEmptyByBackend = function($panelBody) {
+        return $panelBody.find('.no-response').length !== 0;
+    };
+
+    var displayAsEmptyPanel = function($panelBody) {
+        $panelBody.parent().parent().removeClass('panel-info').addClass('panel-default');
```

Comment:
Removed the .parent chain and using .parents(selector) instead. This finds the first ancestor which matches the selector and will work correctly as long as the panel body is inside an element with classes 'panel' and 'panel-info'.